wp-content/plugins/goonj-blocks/src/render.php

```diff
@@ -39,6 +41,14 @@
 	$action_target['id']
 );
 
+function format_date($date) {
```

Comment:
These are global function so we should always prefix with plugin slug
```suggestion
function gb_format_date($date) {
```

Reply:
sure